target/linux/ar71xx/config-4.4

```diff
@@ -47,6 +47,7 @@ CONFIG_ATH79_MACH_AP136=y
 CONFIG_ATH79_MACH_AP143=y
 CONFIG_ATH79_MACH_AP147=y
 CONFIG_ATH79_MACH_AP152=y
+CONFIG_ATH79_MACH_AP531B0=y
```

Comment:
> i will do all change under linux env next time

Yes, please!

>  yes ,i have built firmware with my change, and testted firmware upgrade both via uboot and web

If upgrade over vendor web gui is possible, please include this information in the commit message.